scripts/lib.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
```

Comment:
This file is basically copied from the monorepo (ci/lib.sh). There's a copy in `enterprise-helm` too (`scripts/lib.sh`). Kind of unfortunate to have this script replicate like that, but I can't think of anything better